redirects.yaml

```diff
@@ -339,6 +339,12 @@ redirects:
     destination: https://developer.chrome.com/articles/periodic-background-sync/
   - source: /eyedropper
     destination: https://developer.chrome.com/articles/eyedropper/
+  - source: /web-share-target
+    destination: https://developer.chrome.com/articles/web-share-target/
+  - source: /local-fonts
+    destination: https://developer.chrome.com/articles/web-share-target/
+  - source: /notification-triggers
+    destination: https://developer.chrome.com/docs/web-platform/notification-triggers/
```

Comment:
As I can see this article has translations but the destination doesn't have links to translated versions. Or how does locale selection work on dcc?

Reply:
I think this covers the translated versions since it's so high up in the directory